license-update

```diff
@@ -3,6 +3,10 @@
 license=$(sed -e 's|^#*|// |' <LICENSE)
 
 function _process {
+  if [ ! -f "$FILE" ]
+  then
+    return 0
+  fi
```

Comment:
Package lock and json fix are clear, but what is those lines and user select scene deleted

Reply:
@ivanovSPvirtru These lines handle when a file is removed. The script will pick up a change to the file, but the file no longer exists and can't add a license header.

Reply:
Regarding the `UserScene` file I don't believe they were used. Checking.

Reply:
Confirmed: https://github.com/virtru/protect-and-track/search?q=userselect&unscoped_q=userselect

Reply:
Gotcha!